tests/console/docker_image.pm

```diff
@@ -18,26 +18,45 @@ use strict;
 sub run {
     select_console 'root-console';
 
-    check_var('VERSION', 'Tumbleweed') || die 'Only have docker images for Tumbleweed';
+    my $version = get_required_var('VERSION');
 
-    my $rpm_name   = 'opensuse-tumbleweed-image';
-    my $image_name = 'opensuse/tumbleweed:current';
+    my $image_name;
+    my $image_path;
+
+    if ($version eq 'Tumbleweed') {
```

Comment:
Sorry for yet another round of feedback, but could you please use is_tumbleweed here (no new verification run required).